### ISSUE TITLE
Add prettier?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/shield-studies-addon-template#38]
Not sure if we care about auto-formatting code. It feels a bit excessive in places, and we have some places which may need tweakin'.

```json
"format": "prettier '**/*.{css,js,json}' --trailing-comma=all --ignore-path=.eslintignore --write",
```

Since we'd be using a nearly identical list of ignores for .prettierignore and .eslintignore, I just stuck with one, but not sure we want to auto-format the package.json file.

**Comment by motin** [comment]
+1

**Comment by motin** [comment]
@pdehaan I played around with this a bit, but it seems that prettier is so opinionated that it results in eslint errors. If you find a working combination of prettier and eslint rules, I'm all for using prettier (or some other formatter)

**Comment by motin** [comment]
Actually, https://github.com/prettier/prettier-eslint is based on running prettier + eslint --fix and that fixes the eslint issues, so [here](https://github.com/mozilla/shield-studies-addon-template/pull/49/commits/5f78500820a35c8486976323295e4fded17ea5f9) is a commit that adds npm run format using both prettier and eslint --fix

**Comment by motin** [comment]
This has been addressed in #73
